syntax = "proto3";

package exercise;

message Payload {

}

message Card {
    string id = 1;
    string title = 2;
    Payload payload = 3;
}

message Deck {
    string userId = 1;
    string deckId = 2;
    string description = 3;
    map<string, Card> cards = 4;
    int32 pageToken = 5;
    int32 nextPageToken = 6;
}